### ISSUE TITLE
Can't access headers, returned by fixtures response

### ISSUE THREAD

**Issue by Krakabek** [donejs/donejs#680]
Assume I have fixture like that

``` javascript
fixture("POST /auth", function (request, response) {
    response(200,'success', {foo:'bar'},{baz: 'qux'});
})
```

If I call `xhr.getAllResponseHeaders()` in success callback I get only `Object {Content-Type: "application/json", Accept: "application/json, text/javascript, */*; q=0.01"}`, and nothing about my `baz` header

**Comment by matthewp** [comment]
Thanks, will take a look.

**Comment by matthewp** [comment]
Fixed in can-fixture 0.3.1
